Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/FragmentExtensions.kt

```diff
@@ -23,6 +23,9 @@ fun Fragment.popBackStack(): Boolean {
         }
         return false
     }
+    if (findNavController().backQueue.isEmpty()) {
```

Comment:
Um I think the actual issue lays  in this part in details fragments 
```kotlin
                        if (args.numberOfCertificates == 1) {
                            doNavigate(
                                VaccinationDetailsFragmentDirections
                                    .actionVaccinationDetailsFragmentToPersonOverviewFragment()
                            )
                        } else popBackStack()

in this case the certificate count is `1` the moment the details screen is opened. the action `action_vaccinationDetailsFragment_to_personOverviewFragment` will execute  which also pops up to `personOverviewFragment` inclusively. using `popBackStack` would fix it ,but I'm not sure why the Navigation was done that way. Maybe you can ;)?
```

Reply:
Yes, I think you're right. I will try and see if I can fix it somehow, I had introduced this piece of code when fixing a bug in the past where deleting the last certificate would sometimes cause the app to crash instead of going back to the overview screen, but I know we've made improvements to the certificates flow in the meantime and that might also fix that bug.

Reply:
I've tested this and it's not the cause of the bug... I will get to addressing @chiljamgossow comment and I still think it's the only viable solution for now. Either way, I've only seen it happen if you do these steps that are almost never going to be done in real life. You would have to scan a certificate, delete it right away, and then once again scan a certificate and delete it right away...

Reply:
Did you try to use `popBackStack` when number of certificates is 1 ? I did try it and did not encounter the issue anymore

Reply:
But args.numberOfCertificates will only have the value 1 if you come from `PersonDetailsFragment` and there's one certificate left... If you come from the UQS to the details screen, number of certificates will be 0, even though you've scanned only 1 certificate.